android/src/main/kotlin/com/drenther/upi_pay/UpiPayPlugin.kt

```diff
@@ -64,8 +69,29 @@ class UpiPayPlugin internal constructor(registrar: Registrar, channel: MethodCha
             if (tn != null) {
                 uriBuilder.appendQueryParameter("tn", tn)
             }
+            uriBuilder.appendQueryParameter("mode", "00")
+            uriBuilder.appendQueryParameter("orgid", "000000")
+            val uriByBuilder = uriBuilder.build()
+            Log.d("upi_pay", "initiateTransaction URI (by builder): " + uriByBuilder.toString())
+            */
```

Comment:
Hey @reeteshranjan , it will be great if we can remove the older code and Log statements from the code, and just keep your reasoning comments. That provides sufficient context.

Reply:
Ok thanks. Will add a commit.

Reply:
Taken care in latest commits